src/function.rs

```diff
@@ -0,0 +1,563 @@
+use core::{cmp::Ordering, iter};
+
+use alloc::{borrow::Cow, boxed::Box};
+
+use crate::demangle_auto;
+use crate::lazy::LazyCell;
+use crate::maybe_small;
+use crate::maybe_small::Vec;
+use crate::Error;
+use crate::RangeAttributes;
+use crate::ResDwarf;
```

Comment:
Sorry, by nested import I meant using `::` within the `{}`. So please change to:
```
use alloc::borrow::Cow;
use alloc::boxed::Box;
use alloc::vec::Vec;
use core::cmp::Ordering;
use core::iter;

use crate::{demangle_auto, Error, RangeAttributes, ResDwarf};
use crate::lazy::LazyCell;
use crate::maybe_small;
```

(For this specific case, nested imports wouldn't be bad, but in general I find them harder to read.)